htdocs/fourn/facture/list.php

```diff
@@ -850,6 +851,11 @@
 			$totalpay = $paiement + $totalcreditnotes + $totaldeposits;
 			$remaintopay = $obj->total_ttc - $totalpay;
 
+            //If invoice has been converted and the conversion has been used, we dont have remain to pay on invoice
+            if($facturestatic->type == FactureFournisseur::TYPE_CREDIT_NOTE && $facturestatic->statut == FactureFournisseur::STATUS_CLOSED) {
+                if($facturestatic->isCreditNoteUsed())$remaintopay=0;
```

Comment:
When a credit note of 100 euros is converted as a "credit available". The credit note can be consumed for 20 euros in invoice B, 30 euros in invoice C and having 50 euros not yet consumed (the initial line of 100 euros can be split into 3 lines).
So a better way here to get the remain to pay of a credit note is to add, just after the 

$remaintopay = $obj->total_ttc - $totalpay;

this kind of code :

if ($facturestatic->type == FactureFournisseur::TYPE_CREDIT_NOTE)
{
        $remaintopay = $remaintopay - $this->amountFromThisSupplierCreditNoteNotYetConsumed();
}